include/IECoreHoudini/LiveScene.h

```diff
@@ -137,10 +138,14 @@ class LiveScene : public IECore::SceneInterface
 		// The functions will be called during hasTag and readTags.
 		// readTags will return the union of all custom ReadTagsFns.
 		static void registerCustomTags( HasTagFn hasFn, ReadTagsFn readFn );
-		
+
+
+	protected :
+		virtual LiveScenePtr create() const;
+		virtual LiveScenePtr duplicate( const UT_String &nodePath, const Path &contentPath, const Path &rootPath, double defaultTime, DetailSplitter *splitter ) const;
```

Comment:
I kind of wish we had a better name than `duplicate()`, though I can't think of one... Does `IECoreMaya::LiveScene` have an equivalent method?